code/controllers/configuration.dm

```diff
@@ -79,6 +79,8 @@ GLOBAL_LIST_EMPTY(storyteller_cache)
 	var/allow_drone_spawn = 1				//assuming the admin allow them to.
 	var/drone_build_time = 1200				//A drone will become available every X ticks since last drone spawn. Default is 2 minutes.
 
+	var/enable_mob_sleep = 0  //Experimental - make mobs sleep when no danger is present
+
```

Comment:
I think that should default to on. We can always disable it later if it breaks stuff.

Reply:
It's defaulted to on in config file, so, until it 100% tested on an actual launch, I'd hesitate defaulting it to active.

Reply:
The example config file, not actual server one, which we can easily forgot to edit (chances are that we still don't have access on next hosting for this). No use to test it partially, turn it on always for now.